evaluator/builtin_time_test.go

```diff
@@ -350,6 +350,52 @@ func (s *testEvaluatorSuite) TestSysDate(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testEvaluatorSuite) TestFromUnixTime(c *C) {
+	defer testleak.AfterTest(c)()
+
+	tbl := []struct {
+		isDecimal      bool
+		integralPart   int64
+		fractionalPart int64
+		decimal        float64
+		format         string
+		ansLen         int
+	}{
+		{false, 1451606400, 0, 0, "", 19},
+		{true, 1451606400, 123456000, 1451606400.123456, "", 26},
+		{true, 1451606400, 999999000, 1451606400.999999, "", 26},
+		{true, 1451606400, 999999900, 1451606400.9999999, "", 19},
+		{false, 1451606400, 0, 0, "%Y %D %M %h:%i:%s %x", 19},
+		{true, 1451606400, 123456000, 1451606400.123456, "%Y %D %M %h:%i:%s %x", 26},
+		{true, 1451606400, 999999000, 1451606400.999999, "%Y %D %M %h:%i:%s %x", 26},
+		{true, 1451606400, 999999900, 1451606400.9999999, "%Y %D %M %h:%i:%s %x", 19},
```

Comment:
ok
when pass a decimal input parameter and get the right answer, the mysql.ErrTruncated case would be tested at the same time.